linux_os/guide/system/accounts/accounts-physical/require_singleuser_auth/ansible/shared.yml

```diff
@@ -0,0 +1,25 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+{{% if init_system == "systemd" -%}}
+- name: require single user mode password
+  lineinfile:
+      create: yes
+      dest: /usr/lib/systemd/system/rescue.service
+      regexp: "^#?ExecStart="
+      {{% if product == "fedora" -%}}
```

Comment:
On my RHEL 8.0.0 machine, `/usr/lib/systemd/system/rescue.service` contains `ExecStart=-/usr/lib/systemd/systemd-sulogin-shell rescue`. So I think it isn't applicable only to Fedora but the condition should be extended also to RHEL8.

Reply:
I have realized that if we change this condition we need to change the condition also in Bash, OVAL and rule.yml, because everything that is related to the rule should consistently check or modify the same values.

Reply:
I have filed https://github.com/ComplianceAsCode/content/pull/4407 to address this issue.